ext/mysqli/mysqli.c

```diff
@@ -281,17 +281,19 @@ static void mysqli_warning_free_storage(zend_object *object)
 /* }}} */
 
 /* {{{ mysqli_read_na */
-static zval *mysqli_read_na(mysqli_object *obj, zval *retval)
+static int mysqli_read_na(mysqli_object *obj, zval *retval, zend_bool quiet)
 {
 	zend_throw_error(NULL, "Cannot read property");
```

Comment:
Should check `quiet` here, though it doesn't look like this code can be hit.

Reply:
OK, will do. I only avoided checking for `quiet` because if this code is hit, then it's really a problem.